src/client/visualizations/heat-map/heat-map.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2015-2016 Imply Data, Inc.
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// For some reason tsc compiler does not see this file.
+// Remove when issue is identified.
+// tslint:disable-next-line: no-reference
+/// <reference path="../../index.d.ts" />
+
+import { TooltipProps, TooltipWithBounds, withTooltip } from "@vx/tooltip";
+import memoize = require("memoizee");
```

Comment:
looks weird :D

Reply:
proper TypeScript :D

Reply:
Other imports didn't work

Reply:
It's not critical for now, but we need to check why.

Reply:
https://www.typescriptlang.org/docs/handbook/modules.html - older packages for compatibility. It looks weird, but it's 100% ok to use.